src/storage-preview/azext_storage_preview/_validators.py

```diff
@@ -286,6 +286,10 @@ def validator(cmd, namespace):
             if guess_from_file:
                 new_props = guess_content_type(ns[guess_from_file], new_props, settings_class)
 
+        if process_md5:
+            from .track2_util import string_to_bytes
+            new_props.content_md5 = string_to_bytes(new_props.content_md5)
```

Comment:
In model, content md5 is string as shown below:
https://github.com/Azure/azure-sdk-for-python/blob/00c9044986e1bb35f7d2be0deb5935b5342ae1a7/sdk/storage/azure-storage-blob/azure/storage/blob/_models.py#L621
So I would assume SDK could handle it. Let's wait for the bug fix in sdk. https://github.com/Azure/azure-sdk-for-python/issues/15919

Reply:
OK, Maybe we can merge this compatible logic first, and then delete the logic after SDK fixes the problem